code/game/objects/items/candle.dm

```diff
@@ -63,6 +63,11 @@
 		update_icon()
 		set_light(0)
 
+/obj/item/candle/bullet_act(var/obj/item/projectile/Proj)
+	if(istype(Proj ,/obj/item/projectile/beam))
+		if(!istype(Proj ,/obj/item/projectile/beam/lasertag) && !istype(Proj ,/obj/item/projectile/beam/practice) && !Proj.nodamage)
```

Comment:
Don't the lasertag and practice beams already have nodamage?

Reply:
strangely enough, it doesn't seem like it

Reply:
nodamage is only used for stuns IIRC
But it seems easier to just check if damage is 0

Reply:
makes sense, updated that check